qpc/utils.py

```diff
@@ -369,7 +369,9 @@ def setup_logging(verbosity):
         # (at least until we add an option controlling the log format)
         stream_handler.setFormatter(logging.Formatter(log_fmt))
     stream_handler.setLevel(log_level)
-    logger.addHandler(stream_handler)
+    main_package_name, _, _ = __name__.partition('.')
```

Comment:
```suggestion
    main_package_name, *_ = __name__.partition('.')
```
Since we don't care for the remainder partitions, I suggest changing that.